CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 * **New Resource**: `tfe_workspace_variable_set` ([#537](https://github.com/hashicorp/terraform-provider-tfe/pull/537)) adds the ability to assign a variable set to a workspace in a single, flexible resource.
 * r/tfe_workspace, d/tfe_workspace: `trigger-patterns` ([#502](https://github.com/hashicorp/terraform-provider-tfe/pull/502)) attribute is introduced to support specifying a set of [glob patterns](https://www.terraform.io/cloud-docs/workspaces/settings/vcs#glob-patterns-for-automatic-run-triggering) for automatic VCS run triggering.
+* r/organization: Add `workspace_limit` setting ([#521](https://github.com/hashicorp/terraform-provider-tfe/pull/521))
```

Comment:
Do we want to mention that this is available in TFE only?

Reply:
I can do that 🙇 I did mention it was TFE only in the docs at least!